src/generators/euclideangraphs.jl

```diff
@@ -35,23 +35,23 @@ function euclidean_graph(points::Matrix;
     d, N = size(points)
     g = Graph(N)
     weights = Dict{Edge,Float64}()
-    cutoff < 0. && (cutoff=typemax(Float64))
+    cutoff < 0. && (cutoff = typemax(Float64))
     if bc == :periodic
-        maximum(points) > L &&  error("Some points are outside the box of size $L.")
+        maximum(points) > L && error("Some points are outside the box of size $L.")
     end
-    for i=1:N
-        for j=i+1:N
+    for i = 1:N
+        for j = (i + 1):N
```

Comment:
It is, but it's explicit, as you said. There is a fundamental difference between
```
for j = x : y + z
```
and
```
for j = (x : y) + z
```
and I'm trying to make sure we don't get bitten by ambiguities.